qf_lib/data_providers/bloomberg_beap_hapi/bloomberg_beap_hapi_data_provider.py

```diff
@@ -147,6 +148,8 @@ def get_history(self, tickers: Union[BloombergTicker, Sequence[BloombergTicker]]
             frequency of the data
         universe_creation_time: datetime
             Used only if we want to get previously created universe, fields universe or request
+        currency: Optional[str]
```

Comment:
the method has the parameter currency of type str, but in the docstring it is of type Optional[str]

Reply:
Fixed typings